build/e2e_test.go

```diff
@@ -789,9 +789,12 @@ func TestDeltaShards(t *testing.T) {
 }
 
 // With this test we want to capture regressions in the names returned by our
-// language detection. We rely on the detected language and its spelling, for
-// example, in scoring (see scoreKind).
-func TestDetectLanguage(t *testing.T) {
+// language detection and the scores assigned to file matches. We rely on the
+// detected language and its spelling, for example, in scoring (see scoreKind).
+func TestScoring(t *testing.T) {
+	if checkCTags() == "" {
```

Comment:
a common pattern we do is if we are on CI we fail if a dep like this is missing. You can check if `os.Getenv("CI")` is non-empty.

Reply:
done